### PR TITLE
feat: add Html tables

### PR DESCRIPTION
**Pull request by FrancisDeh**
### First PR
This pr adds a table to our _html_ **entities**.
examples like `<html></html>`

> this is a quote

- [ ] table header
- [ ] table body
- [ ] table footer

### CODE REVIEW

**Review by Fafagrace — APPROVED** [review]
Pull request allowed

File: index.html [hunk 1]
```diff
@@ -44,6 +44,55 @@ <h4>Visit our other pages</h4>
...
+                        <th>#</th>
+                        <th>Members</th>
+                        <th>Favorite Food</th>
+                        <th>Favorite Drink</th>
+                        <th>Actions</th>
+                    </tr>
+                </thead>
+        
+                <tbody>
+                    <tr>
```
Comment (FrancisDeh) [inline index.html:60]: this looks cool

File: index.html [hunk 2]
```diff
@@ -44,6 +44,55 @@ <h4>Visit our other pages</h4>
...
+                        <td>Malt</td>
+                        <td>🗑️</td>
+                    </tr>
+                    <tr>
+                        <td>2</td>
+                        <td>Fafa</td>
+                        <td>Jollof</td>
+                        <td>Sobolo</td>
+                        <td>🗑️</td>
+                    </tr>
```
Comment (FrancisDeh) [inline index.html:73]: change `<td>🗑️</td>` to` <td>x</td>`

Comment (FrancisDeh) [reply index.html:73]: i understand, will make the changes

**Pull request merged**